dashboard/src/app/(authenticated)/externalEventDef/[name]/components/ExternalEventDef.tsx

```diff
@@ -35,12 +35,10 @@ export const ExternalEventDef: FC<Props> = ({ spec }) => {
         tenantId,
         bookmarkAsString: pageParam,
         limit,
-        externalEventDefNameAndStatus: {
-          externalEventDefName: spec.id?.name ?? '',
-          isClaimed,
-        },
-        // earliestStart: createdAfter ? localDateTimeToUTCIsoString(createdAfter) : undefined,
-        // latestStart: createdBefore ? localDateTimeToUTCIsoString(createdBefore) : undefined,
+        externalEventDefId: ExternalEventDefId.create({ name: spec.id?.name }),
```

Comment:
We never use objects like this. Lets use object literals instead to match other code.

Reply:
You're the dashboard guy so I'll take your word for it 😅 But I like this way of writing objects because it is more explicit or in-your-face about where the schema is derived.